services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -364,9 +364,9 @@ export class NOWPermitGeneration extends Component {
                     ) : (
                       <>
                         <NullScreen type="draft-permit" />
-                        <NOWActionWrapper permission={Permission.EDIT_PERMITS}>
-                          <Button onClick={this.startPreDraft}>Start Draft Permit</Button>
-                        </NOWActionWrapper>
+                        {/* <NOWActionWrapper permission={Permission.EDIT_PERMITS}> */}
```

Comment:
Can this commented-out code be removed?

Reply:
@mmumby

Reply:
This will also be resolved in my PR... sorry for all the confusion and WIPS @exzizt